packages/sdk/src/swap/types.ts

```diff
@@ -69,6 +73,7 @@ export interface CommonStatusFields extends ChainsAndAssets {
     | undefined;
 }
 
+// this is not 100% the ACTUAL response type. Its a visual guide as to what you can expect. The real type is inferred from the trpc output schema
```

Comment:
i would be okay with just ditching it

Reply:
i think we probably want it to display in docs.. unless we show zod schema in the docs

Reply:
we generate the type for the docs from the zod schema in other places, so we can do it here too 🙂 its in the `printTypes.mjs` script